<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1131</width>
    <height>636</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>fyp test</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QWidget" name="wgtPlot" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="layoutPlot"/>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QWidget" name="widget_2" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>100</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <widget class="QPushButton" name="btnLoad">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>17</y>
         <width>161</width>
         <height>60</height>
        </rect>
       </property>
       <property name="text">
        <string>load (csv)</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnRoll">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>161</width>
         <height>60</height>
        </rect>
       </property>
       <property name="text">
        <string>load (raw)</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnSave">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>325</y>
         <width>161</width>
         <height>60</height>
        </rect>
       </property>
       <property name="text">
        <string>Save as csv</string>
       </property>
      </widget>
      <widget class="Line" name="dvdSaveLoad">
       <property name="geometry">
        <rect>
         <x>-10</x>
         <y>250</y>
         <width>211</width>
         <height>20</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="lblFrom">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>405</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>From (s) :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="tfFrom">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>425</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="lblTo">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>465</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>To (s) :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="tfTo">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>485</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="lblSaveError">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>282</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Error text</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="lblLoadError">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>185</y>
         <width>161</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="text">
        <string>Error text</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="btnSavePreview">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>530</y>
         <width>161</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Preview</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1131</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
